src/Backend/Modules/Blog/Actions/Add.php

```diff
@@ -125,15 +125,6 @@ private function validateForm()
                 $this->frm->getField('category_id')->addError(BL::err('FieldIsRequired'));
             }
 
-            if ($this->imageIsAllowed) {
```

Comment:
Hmm don't think so, the if-block underneath "validate the image" can be fully removed and you would end up with a blank if-statement that checks if imageIsAllowed?